api/payments.js

```diff
@@ -328,8 +329,9 @@ function getPayment(request, response, next) {
         };
 
         _.extend(result, {
+          client_resource_id: transaction.client_resource_id || '',
           hash: transaction.hash || '',
-          ledger: !_.isUndefined(transaction.inLedger) ? String(transaction.inLedger) : String(transaction.ledger_index),
+          ledger: !transaction.inLedger ? String(transaction.inLedger) : String(transaction.ledger_index),
```

Comment:
Keep in mind that `0` is a valid ledger. I don't think you'll ever see it, but it's better to be correct